app/src/main/java/com/hedvig/app/feature/offer/OfferRepository.kt

```diff
@@ -33,20 +32,26 @@ class OfferRepository(
 ) {
     fun offerQuery(ids: List<String>) = OfferQuery(localeManager.defaultLocale(), ids)
 
-    fun offer(ids: List<String>): SharedFlow<OfferResult> {
+    fun offer(ids: List<String>): Flow<OfferResult> {
         return if (featureManager.isFeatureEnabled(Feature.QUOTE_CART)) {
             val subscription = QuoteCartSubscription(localeManager.defaultLocale(), ids.first())
             apolloClient.subscribe(subscription)
                 .safeSubscription()
                 .map { it.toResult() }
-                .shareIn(CoroutineScope(SupervisorJob()), started = SharingStarted.Lazily)
+                .onStart {
+                    val query = QuoteCartQuery(localeManager.defaultLocale(), ids.first())
```

Comment:
This isn't new to this PR I guess, but doing .first() will simply throw if the list is empty which feels a bit scary. Are we like 100% we're never doing that mistake?
Would it be worth it to use a `NonEmptyList` on the `ids` parameter potentially?